api/v1beta1/placementapi_webhook.go

```diff
@@ -121,18 +126,27 @@ func (r *PlacementAPI) ValidateDelete() (admission.Warnings, error) {
 }
 
 func (r PlacementAPISpec) ValidateCreate(basePath *field.Path) field.ErrorList {
```

Comment:
note to myself: This is called by the webhook within the placement-operator via `func (r *PlacementAPI) ValidateCreate()`